src/locales/de/translation.json

```
@@ -1,185 +1,185 @@
-        "gameplayTools": "Gameplay Tools",
+        "gameplayTools": "Spiel-Tools",
         "setupHelper": "$t(setupHelper.title)",
         "locationTokens": "$t(locationTokens.title)",
         "monsterRoller": "$t(monsterRoller.title)",
         "lostMount": "$t(lostMount.title)",
-        "browseAllTools": "Browse All Tools",
-        "projectHelp": "Project Help",
-        "githubIssues": "Github Issues",
-        "githubDiscussions": "Github Discussions",
+        "browseAllTools": "Entdecke alle Tools",
+        "projectHelp": "Projekt-Hilfe",
+        "githubIssues": "Github Fehler",
+        "githubDiscussions": "Github Diskussionen",
         "projectTimeline": "$t(footer.projectTimeline)",
         "__flag": "🇩🇪"
     },
     "monsterPicker": {
-        "toggle": "Toggle changes:",
-        "sirenExplain": "Siren excluded as it can't be randomly selected",
-        "legendary": "Legendary",
-        "allLegendary": "All Legendary",
-        "title": "Roll A New Monster"
+        "toggle": "Umschalten bewirkt:",
+        "sirenExplain": "Sirene ausgeschlossen, da sie nicht zufällig gewählt werden kann",
+        "legendary": "Legendär",
+        "allLegendary": "Alle Legendären",
+        "title": "Würfel ein neues Monster"
     },
     "monsterRoller": {
-        "title": "Monster Roller"
+        "title": "Monster Würfeln"
     },
     "exps": {
-        "adventurePack": "Adventure Pack",
-        "lostMount": "Lost Mount",
+        "adventurePack": "Abenteuerset",
+        "lostMount": "Verlorenes Reittier",
         "ciri": "Ciri",
-        "wildHunt": "Wild Hunt",
-        "monsterTrail": "Monster Trail",
-        "mages": "Mages",
-        "legendaryHunt": "Legendary Hunt",
+        "wildHunt": "Wilde Jagd",
+        "monsterTrail": "Monsterjagd",
+        "mages": "Zauberinnen und Magier",
+        "legendaryHunt": "Legendäre Monster",
         "skellige": "Skellige",
-        "monsterPack": "Monster Pack",
+        "monsterPack": "Monster Paket",
         "mountedEredin": "$t(monsters.Mounted Eredin)"
     },
     "locationTokens": {
-        "title": "Location Tokens",
-        "mountain": "Mountain",
-        "forest": "Forest",
-        "water": "Water",
+        "title": "Ortsplättchen",
+        "Gebirge": "Mountain",
+        "Wald": "Forest",
+        "Wasser": "Water",
         "Behelt Nar": "Behelt Nar",
         "Kaer Seren": "Kaer Seren",
         "Hengfors": "Hengfors",
         "Kaer Morhen": "Kaer Morhen",
         "Ban Ard": "Ban Ard",
         "Cidaris": "Cidaris",
         "Novigrad": "Novigrad",
-        "Vizima": "Vizima",
+        "Vizima": "Wyzima",
         "Vengerberg": "Vengerberg",
         "Cintra": "Cintra",
         "Haern Caduch": "Haern Caduch",
         "Beauclair": "Beauclair",
         "Glenmore": "Glenmore",
         "Doldeth": "Doldeth",
         "Loc Ichaer": "Loc Ichaer",
-        "Gorthur Guaed": "Gorthur Guaed",
+        "Gorthur Guaed": "Gorthur Gvaed",
```

Small fix from Kaligula:

```suggestion
        "Gorthur Gvaed": "Gorthur Gvaed",
```